Client/Application/LeanplumIntegration.swift

```diff
@@ -124,13 +124,13 @@ class LeanplumIntegration {
     // Events
 
     func track(eventName: LeanplumEventName) {
-        if shouldSendToLP() {
+        if !UIApplication.isInPrivateMode && shouldSendToLP() {
```

Comment:
Nit: You could just bake in the private mode check into the `shouldSendToLP` method so all of the conditions are in one place.